raiden/network/proxies/token_network.py

```diff
@@ -531,8 +652,8 @@ def close(
             self,
             channel_identifier: typing.ChannelID,
             partner: typing.Address,
-            nonce: typing.Nonce,
             balance_hash: typing.BalanceHash,
+            nonce: typing.Nonce,
```

Comment:
why did you change the order of the arguments? this can lead to bugs if not all places are updated.

Reply:
I know ... It's something @err508 started in his first PR and I also commented against doing [here](https://github.com/raiden-network/raiden/pull/2018#discussion_r2070552450).

His reasoning was that he wanted same order as the contract arguments. The code he had introduced had bugs as it had not changed all occurences. I just rolled with it in the end but I made sure that every single occurrence is changed and is also called with kwargs so the order no longer matter.